### PR TITLE
Removed identifier passing from uploadVideo()

### PR DESCRIPTION
**Pull request by dcelik**
This PR removes the identifier argument from uploadVideo as that functionality is no longer supported. See our [SantosCloud PR](https://github.com/santosfamilyfoundation/SantosCloud/pull/69) for the accompanying server side fix.

### CODE REVIEW

**Review by jabb1123 — DISMISSED** [review]

**Review by youralien — DISMISSED** [review]

**Comment by dcelik** [comment]
I added another functionality addition that decides to stream the data if it is over 100MB instead of loading it into memory and then sending it. The PR should be re-reviewed before merging.

File: application/cloud_api.py [hunk 1]
```diff
@@ -68,9 +69,25 @@ def _writeIds(self):
...
+            # incase we need to do streaming
+            files = {'video' : (os.path.basename(video_path), video)}
+
+            #Use a multipartencoder to stream the file data as just data
+            m = MultipartEncoder(fields = files)
+
+            # Get file size in MB's and check if it is greater than a 100 MB
+            # We use a 100MB size limit for transitioning to streaming rather
+            # than loading into memory
+            if m.len/(2**20) >= 100:
```
Comment (youralien) [inline application/cloud_api.py]: I don't understand the math here on first glance. Comment what units m.len is.  Why 2^20?

File: application/cloud_api.py [hunk 2]
```diff
@@ -3,6 +3,7 @@
 import pickle
 import uuid
 import requests
+from requests_toolbelt import MultipartEncoder
```
Comment (youralien) [inline application/cloud_api.py:6]: you should rebase dev (now that Qt5 branch is merged) and add requests_toolbelt to the install scripts

**Review by youralien — CHANGES_REQUESTED** [review]

File: application/cloud_api.py (hunk 1 above)
Comment (youralien) [reply application/cloud_api.py]: Now that I have read your cloud code, I'd prefer seeing the 1024*1024 for MB and then see  something in the comment like `1024*1024 bytes in a MB.`

File: application/cloud_api.py (hunk 2 above)
Comment (dcelik) [reply application/cloud_api.py:6]: Sounds good, ill get on that.

File: application/cloud_api.py (hunk 1 above)
Comment (dcelik) [reply application/cloud_api.py]: Okay, I will revise my comment and math to make it clearer

**Comment by youralien** [comment]
PR's look ugly with all those commits :(

Can't you rebase on dev?

**Comment by dcelik** [comment]
I tried to rebase on dev :(

**Comment by youralien** [comment]
I used a force push and now it looks clean

**Review by youralien — APPROVED** [review]

**Comment by dcelik** [comment]
Thank you!

**Pull request merged**